src/Workspaces/Core/Portable/Shared/Extensions/ITypeSymbolExtensions.UnavailableTypeParameterRemover.cs

```diff
@@ -45,7 +45,7 @@ public override ITypeSymbol VisitNamedType(INamedTypeSymbol symbol)
                 return symbol;
             }
 
-            return symbol.ConstructedFrom.Construct(arguments.ToArray());
+            return symbol.ConstructedFrom.Construct([.. arguments]);
```

Comment:
>([.. arguments]); [](http://example.com/codeflow?start=51&length=17)

Could we just have do ToImmutableArray on the "var arguments = " line above and not needed this?